references/product-recommendations-v1/apiproxy/policies/AM-copy-false.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
License headers will be required on all .xml files

Reply:
Updated all xml, js and sh files